### PR TITLE
Write capture skill

### PR DESCRIPTION
**Pull request by Yudai-8-om**
## Description
capture skill

tick returns the list of actions the robot can take.
is_done function is copied from the action.

### CODE REVIEW

File: rj_gameplay/rj_gameplay/skill/capture.py [hunk 1]
```diff
@@ -30,6 +30,9 @@ def __init__(self):
         self.root = self.capture_behavior
         self.root.setup_with_descendants()
 
-    def tick(self, world_state: rc.WorldState, robot: rc.Robot) -> None:
+    def tick(self, robot: rc.Robot, world_state: rc.WorldState) -> None:
+    	self.robot = robot
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/capture.py:34]: indent issue, should match the line below

File: rj_gameplay/rj_gameplay/skill/capture.py [hunk 2]
```diff
@@ -30,6 +30,9 @@ def __init__(self):
         self.root = self.capture_behavior
         self.root.setup_with_descendants()
 
-    def tick(self, world_state: rc.WorldState, robot: rc.Robot) -> None:
+    def tick(self, robot: rc.Robot, world_state: rc.WorldState) -> None:
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/capture.py]: this method should return a list of actions, see `move.py` or `mark.py` on the ros2 branch

File: rj_gameplay/rj_gameplay/skill/capture.py [hunk 3]
```diff
@@ -30,6 +30,9 @@ def __init__(self):
...
+    	self.robot = robot
         self.root.tick_once(robot, world_state)
-        # TODO: change so this properly returns the actions intent messages
+
+    def is_done(self, world_state) -> bool:
+    	self.capture.is_done(world_state)
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/capture.py]: this should return a bool, again see the move or mark skills

**Review by kfu02 — CHANGES_REQUESTED** [review]
You have a few minor changes to make. Nice job getting the git stuff sorted, though.

**Review by kfu02 — APPROVED** [review]
Looks good!

**Review by HussainGynai — APPROVED** [review]
LGTM

**Pull request merged**